### ISSUE TITLE
Editor doesn't work on Windows 11

### ISSUE THREAD

**Issue by Brickticks** [goatfungus/NMSSaveEditor#918]
Simple as that, this editor doesn't work on Windows 11.  It pulls up what appears to be a Command Prompt window that just keeps showing the same message over and over and over, filling the page with said message and never actually opening the editor.  Also, given how fast it keeps writing out this message, I can't tell what it says.  Rockatoa, Brickticks out!

**Comment by davemager** [comment]
I can assure it you works on Windows 11 - it does for me. Make sure your version of Java runtime environment is updated (I currently am running Version 8 Update 401 (build 1.8_401-b10)). It doesn't yet work for any of the saves touched/created by the new Experimental build on Steam, yet. We'll have to wait patiently for the new save format to be incorporated here.

**Comment by spideruk07** [comment]
yup it definatly works with win 11, like dave said check your java and if on experimental or new update then you will have to wait for an update

**Comment by MGJerry** [comment]
Def working on Win 11, usually it's caused by java runtime and you just need to update it

**Comment by RTXonFPSoff** [comment]
It always worked for me on Windows 11. This issue should be closed and marked as user error. The guy maintaining this awesome program has other issues to look after.

**Comment by fraronk** [comment]
I had the same issue, actually i reinstalled Java version 8 Update 401 (build 1.8.0_401-b10) from [https://www.java.com/it/download/manual.jsp](https://www.java.com/it/download/manual.jsp) and it improved the situation.
But i can still notice some graphic glitches, at least the editor is usable.

**Comment by fraronk** [comment]
![image](https://github.com/goatfungus/NMSSaveEditor/assets/23724437/0e87f0c9-c9c9-40b6-aac1-b6fcd470d69f)
nevermind it is still like this
